pkg/kubectl/cmd/util/openapi/validation/validation_test.go

```diff
@@ -335,4 +335,63 @@ items:
 
 		Expect(err).To(BeNil())
 	})
+
+	It("fails because apiVersion is not provided", func() {
+		err := validator.ValidateBytes([]byte(`
+kind: Pod
+metadata:
+  name: name
+spec:
+  containers:
+  - name: name
+    image: image
+`))
+		Expect(err.Error()).To(Equal("apiVersion not set"))
+	})
+
+	It("fails because apiVersion type is not string and kind is not provided", func() {
+		err := validator.ValidateBytes([]byte(`
+apiVersion: 1
+metadata:
+  name: name
+spec:
+  containers:
+  - name: name
+    image: image
+`))
+		Expect(err.Error()).To(Equal("[apiVersion isn't string type, kind not set]"))
+	})
+
+	It("fails because List first item is missing kind and second item is missing apiVersion", func() {
+		err := validator.ValidateBytes([]byte(`
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  metadata:
+    name: name
+  spec:
+    replicas: 1
+    template:
+      metadata:
+        labels:
+          name: name
+      spec:
+        containers:
+        - name: name
+          image: image
+- kind: Service
+  metadata:
+    name: name
+  spec:
+    type: NodePort
+    ports:
+    - port: 123
+      targetPort: 1234
+      name: name
+    selector:
+      name: name
+`))
```

Comment:
you don't have a test for the feature you just implemented, do you?

Reply:
@apelisse hi, actually yes I do. The last two tests here both shows failure with aggregated errors are towards the features we recently merged. i.e. "([apiVersion isn't string type, kind not set]"  and "[kind not set, apiVersion not set]") ] .. LOC 352 and 396. Hope makes sense. Thanks!

Reply:
But it happens in two different objects. I think that test would have worked without your change? Maybe I'm wrong though.

Reply:
OK, found it 🤣

Reply:
LOL Thanks!!